lib/cn/cn.c

```diff
@@ -1099,17 +1094,27 @@ cn_open(
     cn->cp = kvdb_kvs_cparams(kvs);
     cn->cn_cndb = cndb;
     cn->ikvdb = ikvdb_kvdb_handle(kvdb_kvs_parent(kvs));
-    cn->cn_dataset = ds;
+    cn->cn_dataset = mp;
     cn->cn_cnid = cnid;
     cn->cn_cflags = kvdb_kvs_flags(kvs);
     cn->cn_kvdb_health = health;
     cn->cn_mpool_props = mpprops;
 
-    staging_absent = mpool_mclass_props_get(ds, MP_MED_STAGING, NULL);
-    if (staging_absent) {
-        if (strcmp(rp->mclass_policy, "capacity_only")) {
-            log_warn("Staging media not configured, using capacity_only media class policy");
-            strlcpy(rp->mclass_policy, "capacity_only", HSE_MPOLICY_NAME_LEN_MAX);
```

Comment:
We got rid of both types of fallback?  The mblock allocation fallback and the policy fallback?  That's goodness.  Not sure what the value of this was.

Reply:
Yes, that's correct.